app/javascript/blogging/index.js

```diff
@@ -0,0 +1,2 @@
+export { PopupOpenerController } from './popup_opener_controller'
+
```

Comment:
eliminar linea en blanco

```suggestion
export { PopupOpenerController } from './popup_opener_controller'
```

Reply:
Ya mandé el commit, me parece que te referías a que habían 2 líneas en blanco al final